platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -8,16 +8,22 @@ supervisorctl start rsyslogd
 
 supervisorctl start syncd
 
+while true; do
+    if [ -e /var/run/sswsyncd/sswsyncd.socket ]; then
+        break
+    fi
+    sleep 1
+done
+
 # If this platform has an initialization file for the Broadcom LED microprocessor, load it
 if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
     # Wait until syncd has created the socket for bcmcmd to connect to
```

Comment:
@qiluo-msft, I don't quite understand what you meant here~  Could you do me a favor to be specific it more?